tooling/lib/test/karma.js

```diff
@@ -25,31 +25,22 @@ exports.test = async function test(options, packageName, files) {
   }
 
   if (options.xunit || process.env.COVERAGE || process.env.CIRCLECI || process.env.CI) {
-    for (let i = 0; i < 3; i += 1) {
```

Comment:
Why did you remove the 'retry 3 times'?

Reply:
This is old code I'm guessing that was setup for Jenkins and relative flakiness from back when it was probably necessary to retry.
Results are more reliable now and could give the possibility of testing against more "trickle-down" packages instead of only modified packages